packages/react-native/ReactCommon/react/renderer/components/view/BaseTouch.cpp

```diff
@@ -23,13 +23,6 @@ void setTouchPayloadOnObject(
   object.setProperty(runtime, "target", touch.target);
   object.setProperty(runtime, "timestamp", touch.timestamp * 1000);
   object.setProperty(runtime, "force", touch.force);
-#if TARGET_OS_OSX // [macOS
-  object.setProperty(runtime, "button", touch.button);
-  object.setProperty(runtime, "altKey", touch.altKey);
-  object.setProperty(runtime, "ctrlKey", touch.ctrlKey);
-  object.setProperty(runtime, "shiftKey", touch.shiftKey);
-  object.setProperty(runtime, "metaKey", touch.metaKey);
-#endif // macOS]
```

Comment:
We may want to undo this, as it is useful to know if Ctrl+Click happened vs a normal click. I'l probably separate that to another PR.